lib/oxidized/model/vyatta.rb

```diff
@@ -3,7 +3,7 @@ class Vyatta < Oxidized::Model
 
   # Brocade Vyatta / VyOS model #
 
-  prompt /@.*?:~\$\s/
+  prompt /@.*(:~\$|>)/
```

Comment:
Why has the trailing whitespace (\s) gone?

Reply:
my bad, that was an oversight in my testing, updated.